models/mpas_atm/model_mod.f90

```diff
@@ -5414,8 +5508,8 @@ subroutine find_pressure_bounds(state_handle, ens_size, p, cellid, nbounds, &
 where(ier(:) == 0) ier(:) = temp_ier(:)
 
 ! Check for out of the column range
-where(p(:) < pressure(nbounds, :)) ier(:) = 81
-!where(p(:) > pressure(      1, :)) ier(:) = 80
+where(p(:) < pressure(nbounds, :)) ier(:) = 81   ! too high
+!where(p(:) > pressure(      1, :)) ier(:) = 80  ! too low -> just take the lowest level
```

Comment:
Does this mean error code of 80 is never used ... should it not exist?

Reply:
I don't know

Reply:
not a deal breaker